src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const marriend = sex === 'f'
+    ? `${partnerName} is my husband`
+    : `${partnerName} is my wife`;
+
+  const partner = isMarried
+    ? marriend
+    : 'I am not married';
```

Comment:
You will avoid repetition this way
```suggestion
  const marriend = sex === 'f'
    ?  husband`
    :  wife`;

  const partner = isMarried
    ? `${partnerName} is my ${married}
    : 'I am not married';
```